docs/cloud-tutorials/cloud-cli.md

```diff
@@ -321,6 +321,28 @@ It is meant to be used with [`database restore`](#dbos-cloud-db-restore) during
 
 ---
 
+### `dbos-cloud app secrets create`
+
+**Description:**
+Create a new secret associated with an application, or update an existing secret.
```

Comment:
Similar comment as above. It'd be great to clarify the behavior of secret updates.